### PR TITLE
Inter-Community navigation 

### PR DESCRIPTION
**Pull request by salman-neslit**
## Link to Issue
Closes: #7989

## Description of Changes
- Display communities the user is a member of in the sidebar on the left-hand side.
- Highlight the community icon with a blue circle and make it circular when the user is within the community.
- Display the community icon in a square format when the user is not within the community.
- Indicate unread content notifications communities.
- Hide community icons when the user is signed out.

## Test Plan
  - Sidebar visibility of joined communities.
  - Correct icon highlighting when entering and leaving communities.
  - Notifications for unread content in active communities.
     - You can send notification using Knock test workflows.
  - No community icons visible when signed out.

## Deployment Plan

1. 

## Other Considerations

-

### CODE REVIEW

**Review by masvelio — COMMENTED** [review]
image

have communities duplicated

**Comment by salman-neslit** [comment]
> image
> 
> have communities duplicated

@masvelio  not really top one is the fav community section

 Actions
---- (divider)
Favs
---- (divider)
All

File: packages/commonwealth/client/scripts/views/components/sidebar/SidebarNotificationIcon.tsx [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+import './SidebarNotificationIcon.scss';
+
+type SideBarNotificationIconProps = {
+  unreadCount: number;
+};
+
+export const SideBarNotificationIcon = ({
+  unreadCount,
+}: SideBarNotificationIconProps) => (
+  <div className="sidebar-notification-icon">
```
Comment (masvelio) [inline packages/commonwealth/client/scripts/views/components/sidebar/SidebarNotificationIcon.tsx:12]:
```suggestion
  <div className="SideBarNotificationIcon">
```

hey, I mentioned it multiple times, even in this PR

**Comment by masvelio** [comment]
> > image
> > have communities duplicated
> 
> @masvelio not really top one is the fav community section
> 
> Actions ---- (divider) Favs ---- (divider) All

@dillchen is this expected? Does not look good when I was clicking it

**Review by masvelio — APPROVED** [review]

**Review by Israellund — APPROVED** [review]

**Comment by dillchen** [comment]
@salman-neslit sorry I should have been more specific here, it should not be duplicated cc: @masvelio 

Actions ---- (divider) Favs ---- (divider) **All - Minus Favs**

**Pull request merged**